src/daft-plan/src/logical_ops/sink.rs

```diff
@@ -54,6 +55,7 @@ impl Sink {
                     res.push(format!("Compression = {}", compression));
                 }
                 res.push(format!("Root dir = {}", root_dir));
+                res.push(format!("IOConfig = {:?}", io_config));
```

Comment:
favor Display?

Reply:
Oh! I thought that IOConfig didn't implement Display, but turns out it was Option<IOConfig> that didn't implement Display. Fixed.